assets/javascripts/discourse/templates/components/group-assignments-menu-item.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if canAssign}}
   <ul class ='nav-pills'>
     <li>
-      {{#link-to 'group.assignments' group}}
-        {{d-icon "user-plus" class="glyph"}}{{i18n 'discourse_assign.group_assignments'}}{{concat ' (' group.assignment_count concat ')'}}
+      {{#link-to "group.assignments" group.assignment_count}}
```

Comment:
That seems a bit weird. Did you mean to change this? I don't see why we would want to pass the count into the link-to component?